test/parallel/test-bootstrap-modules.js

```diff
@@ -92,7 +92,6 @@ if (common.isMainThread) {
   expectedModules.add('NativeModule internal/streams/state');
   expectedModules.add('NativeModule internal/worker');
   expectedModules.add('NativeModule internal/worker/io');
-  expectedModules.add('NativeModule module');
```

Comment:
This LGTM but if it's more to your liking, another option might be:

```suggestion
  if (require('worker_threads').isMainThread)
    expectedModules.add('NativeModule module');
```

Reply:
If it's expected to be present in the main thread then it should be added to the `if (common.isMainThread) {` part above (these lines are in the `else` part), but since the most recent CI passed it looks like this just isn't loaded anymore and the change is correct.

Reply:
Whoops! Missed the condition. Never mind!